File: VECTOAux/VectoAuxiliaries/DownstreamModules/IM14.vb

Imports TUGraz.VectoCommon.Utils
Imports VectoAuxiliaries.Electrics
Imports VectoAuxiliaries.Pneumatics
Imports VectoAuxiliaries.Hvac

Namespace DownstreamModules
	Public Interface IM14
		ReadOnly Property TotalCycleFCGrams As Gram

		ReadOnly Property TotalCycleFCLitres As Liter
	End Interface
End Namespace

File: VECTOAux/BusAuxiliaries/Interfaces/DownstreamModules/IM14.cs
using TUGraz.VectoCommon.Utils;

namespace TUGraz.VectoCore.BusAuxiliaries.Interfaces.DownstreamModules
{
	public interface IM14 : IAbstractModule
	{
		Kilogram TotalCycleFCGrams { get; }

		Liter TotalCycleFCLitres { get; }
	}
}
